internal/cli/atlas/quickstart/quick_start.go

```diff
@@ -44,10 +43,18 @@ import (
 
 //go:generate mockgen -destination=../../../mocks/mock_quick_start.go -package=mocks github.com/mongodb/mongodb-atlas-cli/internal/cli/atlas/quickstart Flow
 
-const quickstartTemplate = `
-Now you can connect to your Atlas cluster with: mongosh -u %s -p %s %s
+const quickstartTemplateMongoshDetected = `
+MongoDB Shell detected. Connecting to your Atlas cluster:
+$ mongosh -u %s -p %s %s
+`
 
+const quickstartTemplateMongoshNotDetected = `
+MongoDB Shell not detected. To install, open www.mongodb.com/try/download/shell
+[MongoDB Shell (mongosh) is an interactive command line interface to query, update and manage data in the MongoDB database.]
+Once mongosh is installed, you can connect to your Atlas Cluster0 database with:
```

Comment:
A few thoughts:
- Agree this is hard to read, and I think the square brackets make it harder too. Do we need them?
- Do we want to hardcode "Atlas Cluster0" or keep it generic?
- Should we be consistent and refer to it as MongoDB Shell everywhere?

Suggestion below with my preferences, but definitely open!

```suggestion
MongoDB Shell (mongosh) is an interactive command line interface to query, update, and manage data in your MongoDB database.

Once you install the MongoDB Shell, connect to your database with:
```